shared/diveIntoCode/go.md

```diff
@@ -53,4 +53,4 @@ CMD ./resin-go-hello-world
 [resinDockerHub]:https://hub.docker.com/u/resin/
 [docker-ref]:https://docs.docker.com/engine/reference/builder/
 [systemd-link]:https://en.wikipedia.org/wiki/Systemd
-[listOfResinNames]:/devicetypes/
\ No newline at end of file
+[listOfResinNames]:/devicetypes/
```

Comment:
Actually it is (which is good), but for some reason GH doesn't show it, but an email from it does show.